subliminal/subtitle.py

```diff
@@ -276,13 +275,16 @@ def guess_matches(video, guess, partial=False):
     matches = set()
     if isinstance(video, Episode):
         # series
-        if video.series and 'series' in guess and sanitized_string_equal(guess['series'], video.series):
+        if video.series and 'title' in guess and sanitized_string_equal(guess['title'], video.series):
             matches.add('series')
+        # series
+        if video.title and 'episode_title' in guess and sanitized_string_equal(guess['episode_title'], video.title):
+            matches.add('title')
```

Comment:
Yes => https://github.com/guessit-io/guessit/issues/232#issuecomment-154168664

Doesn't really matter though, I've finally make this naming choice because it makes guessit codebase more concise and less confuse.